types/picomatch/lib/picomatch.d.ts

```diff
@@ -0,0 +1,77 @@
+import constantsImport = require('./constants');
+import parseImport = require('./parse');
+import scanImport = require('./scan');
+import { PicomatchOptions as PicomatchOptionsImport } from './picomatch-options';
+
+/**
+ * Creates a matcher function from one or more glob patterns. The
+ * returned function takes a string to match as its first argument,
+ * and returns true if the string is a match. The returned matcher
+ * function also takes a boolean as the second argument that, when true,
+ * returns an object with additional information.
+ *
+ * ```js
+ * const picomatch = require('picomatch');
+ * // picomatch(glob[, options]);
+ *
+ * const isMatch = picomatch('*.!(*a)');
+ * console.log(isMatch('a.a')); //=> false
+ * console.log(isMatch('a.b')); //=> true
+ * ```
+ * @param glob One or more glob patterns.
+ * @return Returns a matcher function.
+ * @api public
+ */
+declare function picomatch(
+    glob: string | string[],
+    options?: picomatch.PicomatchOptions,
```

Comment:
Just use PicomatchOptions here

Reply:
Plain `PicomatchOptions` isn't available here as is. My choices with the current implementation are `PicomatchOptionsImport` or `picomatch.PicomatchOptions`.